docs/manifest.json

```diff
@@ -2427,6 +2427,74 @@
                 ]
               ]
             },
+            {
+              "title": "Vue",
+              "collapse": true,
+              "icon": "vue",
+              "items": [
+                [
+                  { "title": "Overview", "href": "/docs/references/vue/overview" },
+                  {
+                    "title": "Guides",
+                    "items": [
+                      [
+                        {
+                          "title": "Migrating from community SDK",
+                          "href": "/docs/references/vue/migrating-from-vue-community-sdk"
+                        }
+                      ]
+                    ]
+                  },
+                  {
+                    "title": "Client-side Helpers",
+                    "items": [
+                      [
+                        {
+                          "title": "`useUser()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-user"
+                        },
+                        {
+                          "title": "`useClerk()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-clerk"
+                        },
+                        {
+                          "title": "`useAuth()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-auth"
+                        },
+                        {
+                          "title": "`useSignIn()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-sign-in"
+                        },
+                        {
+                          "title": "`useSignUp`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-sign-up"
+                        },
+                        {
+                          "title": "`useSession()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-session"
+                        },
+                        {
+                          "title": "`useSessionList()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-session-list"
+                        },
+                        {
+                          "title": "`useOrganization()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-organization"
+                        }
```

Comment:
PR for the rest of the pages [here](https://github.com/clerk/clerk-docs/pull/1712)